### ISSUE TITLE
Admin Page: error on Traffic tab

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#8725]
```
Warning: Failed prop type: Invalid prop `plan` of type `boolean` supplied to `Banner`, expected `string`.
    in Banner (created by Connect(Banner))
    in Connect(Banner) (created by Connect(Banner))
    in Connect(Banner) (created by Connect(Connect(Banner)))
    in Connect(Connect(Banner)) (created by VerificationServicesComponent)
    in VerificationServicesComponent (created by SettingsForm)
    in SettingsForm (created by Connect(SettingsForm))
    in Connect(SettingsForm) (created by TrafficSettings)
    in div (created by TrafficSettings)
    in TrafficSettings (created by Connect(TrafficSettings))
    in Connect(TrafficSettings) (created by SearchableSettings)
    in div (created by SearchableSettings)
    in SearchableSettings (created by Main)
    in div (created by Main)
    in div (created by Main)
    in div (created by Main)
    in Main (created by withRouter(Main))
    in withRouter(Main) (created by Connect(withRouter(Main)))
    in Connect(withRouter(Main)) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in Provider
    in div admin.js:4865:7
```

cc @oskosk 

Related: #8705

**Comment by zinigor** [comment]
I can't reproduce this, can you describe how you get to this error please?

**Comment by jeherve** [comment]
cc @Viper007Bond

**Comment by oskosk** [comment]
This is being addressed by #8735. Steps to replicate are there in the testing instructions. Thanks!

**Comment by Viper007Bond** [comment]
Confirmed fixed. 👍
